### PR TITLE
[Network] Bump azure-mgmt-network to 11.0.0

### PR DESCRIPTION
**Pull request by haroldrandom**
**Description**  

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Bump azure-mgmt-version

**Comment by jiasli** [comment]
Do have `azure-mgmt-version`? 😆

File: src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export.json [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+  "Langugage": "English",
+  "Settings:BackgroundColor": {
+    "cars": {
+      "Toyota": "Acura"
+    },
+    "ship": [
+      "ship1",
+      "ship2",
+      "ship3"
+    ]
```
Comment (fengzhou-msft) [inline src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export.json:13]: Why do you need to modify this file?

Comment (haroldrandom) [reply src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export.json:13]: don't know, life run modify it automatically.

**Comment by haroldrandom** [comment]
weird thing, the job `Test Extensions Loading` keep failing, but the corresponding run success
image

**Pull request merged**